packages/server/lib/open_project.ts

```diff
@@ -0,0 +1,396 @@
+import _ from 'lodash'
```

Comment:
This is not actually new code - I did `git mv open_project.js open_project.ts`. I'm not sure why GH thinks it's a new file.

I did change it to be TypeScript, and use a class, but the code is the same, except for one important change which I explain [here](https://github.com/cypress-io/cypress/pull/17150/files#r661125034).